index.js

```diff
@@ -17,6 +17,10 @@ function LRU (opts) {
 
 inherits(LRU, events.EventEmitter)
 
+Object.defineProperty(LRU.prototype, 'keys', {
+  get: function () { return Object.keys(this.cache) }
+})
+
```

Comment:
I had it as a function (https://github.com/chriso/lru/pull/22/commits/01fca8c262475a5eec2bb9c553ccbe323af5540f) but thought it made more sense for it to be a property.